examples/ai-core/src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,30 @@
+import { openai } from '@ai-sdk/openai';
+import { generateText } from 'ai';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+import { NodeSDK } from '@opentelemetry/sdk-node';
+import { ConsoleSpanExporter } from '@opentelemetry/sdk-trace-node';
+import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
+
+const sdk = new NodeSDK({
```

Comment:
I'd just recommend installing and using `@vercel/otel` instead of NodeSDK.

Reply:
it's an example for testing. I am planning a separate example with `@vercel/otel`

Reply:
Ok. But then let's definitely add that example. I'd really like to avoid people copy-pasting code that we know already won't work when deployed on Vercel.